platform/src/Utility.js

```diff
@@ -171,4 +173,32 @@ export function urlParamPrivateRepo(){
  */
 export function isAuthenticated(){
     return window.sessionStorage.getItem("isAuthenticated") != null;
-}
\ No newline at end of file
+}
+
+/**
+ * Parses the platform configuration files, YAML and JSON types are supported. 
+ * @param {String} contents the configuration file contents
+ * @param {String} extenstion the configuration file extenstion
+ * @returns the parsed configuration object
+ */
+export function parseConfigFile(contents, extenstion="yml"){
+
```

Comment:
Typo: "extenstion" -> "extension"

Reply:
Corrected in 393b3ec2bc292a452777745e3a254b8dcfba0a99